tempesta_fw/t/functional/helpers/control.py

```diff
@@ -251,11 +267,41 @@ def __init__(self):
         self.stats = tempesta.Stats()
         self.host = tf_cfg.cfg.get('Tempesta', 'hostname')
         self.err_msg = ' '.join(["Can't %s TempestaFW on", self.host])
+        self.bce_counter = DmesgLinesCounter(self.node, "Unable to find a backend server")
+        self.ratelimit_burst = 100000
+        self.old_ratelimit_burst = 10
+        self.ratelimit = 1
+        self.old_ratelimit = 5
+
+    def set_ratelimit(self, rl, rlb):
+        cmd = "echo %i > /proc/sys/kernel/printk_ratelimit_burst" % rlb
+        self.node.run_cmd(cmd, timeout=30,
+                          err_msg=(self.err_msg % 'start'))
+        cmd = "echo %i > /proc/sys/kernel/printk_ratelimit" % rl
```

Comment:
By the way, you use wrong interface for rate limits configuration, you need `/proc/sys/net/core/message_cost` and `/proc/sys/net/core/message_burst` which stands for limits for networking code only.